cedar-policy/src/api.rs

```diff
@@ -1795,6 +1844,67 @@ impl Policy {
     /// If `id` is Some, the policy will be given that Policy Id.
     /// If `id` is None, then "JSON policy" will be used.
     /// The behavior around None may change in the future.
+    ///
+    /// ```
+    /// use cedar_policy::Policy;
+    /// let data = r#"
+    ///        {
+    ///            "effect":"permit",
+    ///            "principal":{
+    ///            "op":"==",
+    ///            "entity":{
+    ///                "type":"User",
+    ///                "id":"bob"
+    ///            }
+    ///            },
+    ///            "action":{
+    ///            "op":"==",
+    ///            "entity":{
+    ///                "type":"Action",
+    ///                "id":"view"
+    ///            }
+    ///            },
+    ///            "resource":{
+    ///            "op":"==",
+    ///            "entity":{
+    ///                "type":"Album",
+    ///                "id":"trip"
+    ///            }
+    ///            },
+    ///            "conditions":[
+    ///            {
+    ///                "kind":"when",
+    ///                "body":{
+    ///                   "<":{
+    ///                        "left":{
+    ///                        ".":{
+    ///                            "left":{
+    ///                                "Var":"principal"
+    ///                            },
+    ///                            "attr":"age"
+    ///                        }
+    ///                        },
+    ///                        "right":{
+    ///                        "Value":18
+    ///                        }
+    ///                    }
+    ///                }
+    ///            }
+    ///            ]
+    ///        }
+    /// "#;
+    /// let v = serde_json::from_str(data).unwrap();
+    /// let policy = Policy::from_json(None, v).unwrap();
+    /// println!("{}", policy);
+    /// // output:
+    /// // permit(
+    /// //    principal == User::"bob",
+    /// //    action == Action::"view",
+    /// //    resource == Album::"trip"
+    /// //  )
+    /// //  when
+    /// //    { principal.age > 18 };
```

Comment:
Rather than using a comment here to show the output, it might be nice to `assert!` that the output matches what you expect. So you could have something like:
```rust
let src = r#"
 permit(
   principal == User::"bob",
   action == Action::"view",
   resource == Album::"trip"
 )
 when
   { principal.age > 18 };"#;
let expected_output = Policy::parse(None, src).unwrap();
assert_eq!(policy, expected_output);
```

*No guarantee that this compiles since I didn't try running it.